### PR TITLE
Alerty

### PR DESCRIPTION
**Pull request by austinjpaul**
- add script that can be used to manually refill slowzones on s3 if/when the lambda fails (has been okay this month, but in dec there were many issues)
- differentiate between "no alerts" and "alerts not found"-- an important distinction!!
- allow alert fetching for 114/6/7

### CODE REVIEW

**Review by nathan-weinberg — APPROVED** [review]
LGTM

File: server/refill_slowzones.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+def do_alerts_exist(d):
+    try:
+        get_alerts(d, ["Red"])
+    except Exception as err:
+        if err.response["Error"]["Code"] == "NoSuchKey":
+            return False
+    return True
+
+
+for d in date_range(START, END):
```
Comment (mathcolo) [inline server/refill_slowzones.py:20]: Should this have a sleep in it?

File: server/refill_slowzones.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import datetime
+
+from chalicelib.parallel import date_range
+from chalicelib.s3_alerts import store_alerts, get_alerts
+
+
+START = datetime.date(2022, 1, 3)
```
Comment (mathcolo) [inline server/refill_slowzones.py:7]: Should these be CLI args?

**Review by mathcolo — APPROVED** [review]
Few questions, but otherwise 👍

**Pull request merged**